tests/SetATableTidyUp.tex

```diff
@@ -1,57 +1,74 @@
+	\item \textbf{Cutlery:} Cutlery and other appliances like carpets and napkins are stored inside a closed cupboard or similar type of furniture (e.g. inside a drawer).
+	\item \textbf{Food:} Food and cleaning utensils are placed close to the table (e.g. in a rack or locker).
+	\item \textbf{Cupboard:} The cupboard used to store the cutlery is located close to the table.
+	\item \textbf{Table:} The table may have some objects on top of it, like a carpet or a plate.
+\end{itemize}
 
-This task takes place in a single room, designated \emph{kitchen,} with a \emph{table,} where the relevant objects are stored, some of then on an open closet and other on a closed closet.
+\subsection{Task}
+\label{sattu:task}
+%\begin{figure}[tbp]
+\begin{figure}[h!]
+	\centering
+	\subfloat[Typical arena]{\label{fig:sattu_table1}\includegraphics[height=46mm]{images/tablesetting1.jpg}} ~ 
+	\subfloat[Typical objects]{\label{fig:sattu_table2}\includegraphics[height=46mm]{images/tablesetting2.jpg}}
+	\caption{Table settings: (a) occidental, and (b) Japanese.}
+	\label{fig:arena}
+\end{figure}
 
-In phase~I, the operator requests the robot to setup the table for a meal. In this request, the operator may specify one variation of the meal, e.g., milk or coffee, bread or cookies. The robot should then set the table using items stored in various places of the kitchen, and afterwards it must clean it up, including small spills on the table. Some of the items are easily accessible (e.g., over the kitchen counter) while others may require the opening of doors (e.g., inside a closet). Furthermore, some of these items may constrain the way they are handled by the robot (e.g., avoid pouring contents of a container). These items must be placed on a table reasonably following social conventions in terms of their positions on the table\footnote{Strict adherence to any social convention is out of the scope of the competition, and thus will not be evaluated.}. Midway during placement, the robot owner may change/adjust the position of some of the items; the robot is expected to handle the situation appropriately.
+\begin{itemize}
+	\item[Part I:] Setting up the table
+	\begin{enumerate}
+		\item \textbf{Fetching command:} The operator requests the robot to set up the table. Optionally, the robot may ask the operator \textit{what has to be served}. The operator replies with the randomly selected (and undisclosed) option.
 
-In phase~II, the operator requests the robot to clean up the table. This includes both returning the placed objects to their original location, and cleaning up dirt and spills using a cleaning cloth. This requires the robot to memorize the location the objects were originally recognized.
+		% I find this unnecessary, the table must be clear already or this test will take forever.
+		\item \textbf{Clearing the table:} The robot must analyze the table and remove all unnecessary objects.
```

Comment:
This seems redundant with Phase II and was not in the original description. I think we can just leave this bit out